src/main/scala/com/simple/jdub/Row.scala

```diff
@@ -306,5 +306,20 @@ class Row(rs: ResultSet) {
     colNames.zip(colValues).toMap
   }
 
-  private def extract[A](f: A): Option[A] = if (rs.wasNull()) None else Some(f)
+  private[this] def extract[A](f: A): Option[A] = if (rs.wasNull()) None else Some(f)
+
+  def array[T: reflect.ClassTag](index: Int): Array[T] = extractArray[T](rs.getArray(index + 1))
+
+  def array[T: reflect.ClassTag](name: String): Array[T] = extractArray[T](rs.getArray(name))
+
+  private[this] def extractArray[T: reflect.ClassTag](sqlArray: java.sql.Array): Array[T] = {
+    if (rs.wasNull()) {
+      Array.empty[T]
```

Comment:
We should return an ```Option[Array[T]]``` to maintain the nullable semantics of the database. As it is, there is no way to tell if the column is null or an empty array.